autozone-poc/src/components/Signup/Signup.js

```diff
@@ -0,0 +1,111 @@
+import React, { useState } from "react";
+import $ from 'jquery';
+import axios from "axios";
+import InputWithLabel from "../../utlis/formComponents/InputWithLabel";
+import "./Signup.css";
+import Button from "../../utlis/button/Button";
+
+//export default class Signin extends Component{
+
+  export default function Signup (props) {
+  // const {register, handleSubmit, errors, watch}=useForm();
+  const [firstname, setFirstName] = useState("");
+  const [lastname, setLastName] = useState("");
+  const [emailid, setEmailid] = useState("");
+  const [password, setPasswords] = useState("");
+  const [confirmpassword, setConfirmPassword] = useState("");
+
+  $(document).ready(function(){
+    $('#confirm_password').on('blur', function(){
+      $('input[required]').each(function(element) {
+        let val = $(this).val();
+        if(val !== password){
+          $(`.confirm_password_test .formErrorMsg`).text("please provide same password for confirm password");
+          $(`.confirm_password_test .formErrorMsg`).show();
+        }  else {
+          $(`.confirm_password_test .formErrorMsg`).hide();
+        }
+      });
+    })
+  });
+
+  const onSubmit = (evt) => {
+    const data = {};
+    evt.preventDefault();
+    data.firstname = firstname;
+    data.lastname = lastname;
+    data.emailid = emailid;
+    data.password = password;
+    axios.post("http://localhost:5000/user/signup", data).then((response) => {
```

Comment:
dont use hard-coded links

Reply:
the code will work with that url:http://localhost:5000/user/signup